build.cake

```diff
@@ -349,9 +352,14 @@ Task("BuildMain")
 Task("BuildTest")
     .IsDependentOn("Setup")
     .IsDependentOn("Restore")
-    .IsDependentOn("BuildMain")
```

Comment:
Why?

Reply:
Actually, it turns out that this creates a small problem in that the host projects get built twice -- once during `BuildTest` and then again during `Publish`, since that task depends on `BuildMain`. Best just to have this depend on `BuildMain` and not build the host projects in `BuildTest.